packages/remix-dev/package.json

```diff
@@ -29,6 +29,7 @@
     "esbuild": "0.14.22",
     "esbuild-plugin-cache": "^0.2.9",
     "execa": "^5.1.1",
+    "esbuild-register": "^3.3.2",
```

Comment:
Is this needed as a dependency?

Reply:
i think it's from a merge conflict, i can remove it

Reply:
On that note, how much trouble would it be to clean up the commit history on this branch? If it takes more than 10 seconds, don't bother.

Reply:
should be pretty easy to do, i'll check after lunch

Reply:
eh my git fu is failing me today